airbyte-integrations/bases/source-acceptance-tests/src/main/java/io/airbyte/integrations/acceptance_tests/source/FileBasedSourceAcceptanceTestDriver.java

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Maybe dumb question: what the reason for calling this `FileBaseSourceAcceptanceTestDriver` instead of `SourceAcceptanceTestDriver`? Do we intent for this to only work with file based test?

Reply:
we've experimented with having language specific bindings (e.g. python bindings) in the past. that's why historically we've namespaced it. if we're happy to forget about those for now (which seems reasonable) then we can probably stop namespacing.